website/src/components/Chat/ChatListBase.tsx

```diff
@@ -91,6 +92,7 @@ export const ChatListBase = memo(function ChatListBase({ allowViews, ...props }:
         {allowViews && (
           <ChatViewSelection w={["full", "auto"]} onChange={(e) => setView(e.target.value as ChatListViewSelection)} />
         )}
+        <HideAllChatButton chatIds={chats.map((chat) => chat.id)} onHide={removeItemFromList}></HideAllChatButton>
```

Comment:
this will always re-render, memo if possible

Reply:
make sense. I missed that. Thank you for pointing out.

Reply:
I have made the change. Please do check.